demos/basic_interp_test.py

```diff
@@ -0,0 +1,61 @@
+import pysgrid
```

Comment:
It would be nice to have this as a notebook (in the notebook directory) alongside the other examples/demostrations that we have there.